metricbeat/module/munin/_meta/fields.yml

```diff
@@ -4,8 +4,10 @@
     Munin node metrics exporter
   release: beta
   fields:
-    - name: munin
-      type: group
+    - name: munin.*
```

Comment:
you can remove the `.*` part, it seems that's how we normally do it in fields.yml, the `timeseries.instance` code will be easier too

Reply:
This looks needed in this case, maybe because I am setting up dynamic templates for two levels now.